logisland-plugins/logisland-common-processors-plugin/src/test/java/com/hurence/logisland/processor/alerting/CheckAlertsTest.java

```diff
@@ -0,0 +1,178 @@
+        runner.setProperty("avib1", "cache(\"cached_id1\").value > 12.0");     // ok
+        runner.setProperty("avib2", "cache(\"cached_id2\").value < 5");        // ok
+        runner.setProperty("avib3", "cache(\"cached_id3\").value < 5");        // ko
+
+        runner.setProperty(ComputeTags.DATASTORE_CLIENT_SERVICE, service.getIdentifier());
+        runner.addControllerService(service.getIdentifier(), service);
+        runner.enableControllerService(service);
+
+        runner.assertValid();
+        runner.run();
+        runner.assertAllInputRecordsProcessed();
+        runner.assertOutputRecordsCount(2);
+        runner.assertOutputErrorCount(0);
+
+        for (Record enriched : runner.getOutputRecords()) {
+            if (enriched.getId().equals("avib1")) {
+                assertEquals(enriched.getField(FieldDictionary.RECORD_VALUE).asString(), "cache(\"cached_id1\").value > 12.0");
+            }
+        }
+    }
+
+
+    @Test
+    public void testCountRules() throws InitializationException {
+
+        // create the controller service and link it to the test processor
+        final DatastoreClientService service = new MockDatastoreService();
+        getCacheRecords().forEach(r -> service.put("test", r, false));
+
+        final TestRunner runner = TestRunners.newTestRunner(CheckAlerts.class);
+        runner.setProperty(CheckAlerts.MAX_CPU_TIME, "100");
+        runner.setProperty(CheckAlerts.MAX_MEMORY, "12800000");
+        runner.setProperty(CheckAlerts.MAX_PREPARED_STATEMENTS, "100");
+        runner.setProperty(CheckAlerts.ALLOw_NO_BRACE, "false");
+        runner.setProperty(CheckAlerts.PROFILE_ACTIVATION_CONDITION, "cache(\"cached_id1\").value > 10.0 && cache(\"cached_id2\").value >= 0");
+        runner.setProperty("avib6", "cache(\"cached_id1\").count > 4");        // ok
+
+        runner.setProperty(ComputeTags.DATASTORE_CLIENT_SERVICE, service.getIdentifier());
+        runner.addControllerService(service.getIdentifier(), service);
+        runner.enableControllerService(service);
+
+        runner.assertValid();
+        runner.run();
+        runner.assertAllInputRecordsProcessed();
+        runner.assertOutputRecordsCount(1);
+        runner.assertOutputErrorCount(0);
+
+    }
+
+    @Test
+    public void testDurationRules() throws InitializationException {
+
+        // create the controller service and link it to the test processor
+        final DatastoreClientService service = new MockDatastoreService();
+        getCacheRecords().forEach(r -> service.put("test", r, false));
+
+        final TestRunner runner = TestRunners.newTestRunner(CheckAlerts.class);
+        runner.setProperty(CheckAlerts.MAX_CPU_TIME, "100");
+        runner.setProperty(CheckAlerts.MAX_MEMORY, "12800000");
+        runner.setProperty(CheckAlerts.MAX_PREPARED_STATEMENTS, "100");
+        runner.setProperty(CheckAlerts.ALLOw_NO_BRACE, "false");
```

Comment:
maybe constant should be all uppercase...to be changed on another PR without blocking this one